src/DynamoCore/Utilities/Configurations.cs

```diff
@@ -230,7 +230,7 @@ public class Configurations
 
         #region Backup
 
-        public static string BackupFileNamePrefix = "recent_new_file_";
+        public static string BackupFileNamePrefix = "recent_new_file";
```

Comment:
Since you're changing this, I'm just curious how we arrived to this name. Shouldn't it be `backup_` instead? How is that a *recent new file*? What was on your mind, @elayabharath? :)